tests/module.py

```diff
@@ -75,6 +107,24 @@ def test_module_wrapper(self):
             self.assertIn(new_key, module_wrapper_vars)
             self.assertIs(module_wrapper_vars[new_key], v)
 
+    def test_force_args(self):
+        x = jn.array([1., 2.])
+        model = ModelWithArg()
+        np.testing.assert_almost_equal(model(x, some_arg=0.0), [1., 2.])
+        np.testing.assert_almost_equal(model(x, some_arg=1.0), [8., 9.])
+        # set forced args
+        model.block1.op1 = objax.ForceArgs(model.block1.op1, some_arg=-1.0)
+        np.testing.assert_almost_equal(model(x, some_arg=0.0), [0., 1.])
+        np.testing.assert_almost_equal(model(x, some_arg=1.0), [6., 7.])
+        # set forced args in a list
+        model.block1.seq[0] = objax.ForceArgs(model.block1.seq[0], some_arg=-1.0)
+        np.testing.assert_almost_equal(model(x, some_arg=0.0), [-1., 0.])
+        np.testing.assert_almost_equal(model(x, some_arg=1.0), [4., 5.])
+        # undo force args
+        objax.ForceArgs.undo(model)
+        np.testing.assert_almost_equal(model(x, some_arg=0.0), [1., 2.])
+        np.testing.assert_almost_equal(model(x, some_arg=1.0), [8., 9.])
+
```

Comment:
Add tests for negative cases with `self.assertRaises` to make sure a `ForceArgs(f)` produces the same errors as `f` when called improperly (there was one example in a previous suggestion).